docs/ci-cd-environment/ubuntu-18.04-image.md

```diff
@@ -195,6 +195,30 @@ Available versions:
 - 2.7.0 and 2.7.1
 - jruby-9.2.11.1
 
+
+### How to install a dependency?
```

Comment:
Rename this to `Installing dependancies with apt package manager`. It is more specific.